cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HBaseTableUtil.java

```diff
@@ -118,7 +120,7 @@ protected boolean isCDAPTable(HTableDescriptor hTableDescriptor) {
     // TODO: Once all system tables are upgraded to have CDAP_VERSION in their descriptor, we can then solely rely on
     // that key being present in the descriptor to identify a CDAP Table
     String tableName = hTableDescriptor.getNameAsString();
-    String value = hTableDescriptor.getValue(Constants.Dataset.CDAP_VERSION);
+    String value = hTableDescriptor.getValue(HBaseTableUtil.CDAP_VERSION);
```

Comment:
nit: shouldn't need to qualify ``CDAP_VERSION`` in this class, right? same for its usages below.